src/frontend/src/components/SearchInput/AuthorSearchInput.jsx

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormControl, TextField } from '@material-ui/core';
 
 function AuthorSearchInput({ text, onChange, classes }) {
   return (
     <>
-      <input
-        className={classes.input}
-        list="search-suggestions"
-        placeholder="How to Get Started in Open Source"
-        inputProps={{ 'aria-label': 'search telescope' }}
-        value={text}
-        onChange={onChange}
-      />
-      <datalist id="search-suggestions"></datalist>
+      <FormControl fullWidth={true}>
```

Comment:
I don't understand why you're removing the `<datalist>`.  Can you not fix the border without adjusting the components?

Reply:
The hover on the border can be "fixed," but the size of the inputs(Author/Post and search field) will be different because of the "native" CSS from Material-UI. 

Material-UI has an autocomplete component to our author autocomplete feature; To it works, we will need to refactor some components. But let's do it when implementing the author autocomplete.